src/ExRam.Gremlinq.Core/Elements/Path.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System.Collections.Generic;
+
+namespace ExRam.Gremlinq.Core.GraphElements
+{
+    public sealed class Path
+    {
+        public string[][]? Labels { get; set; }
+        public IElement[]? Objects { get; set; }
```

Comment:
I'm afraid it's unfortuately more complicated. Consider the query g.V().out().map(count()).path(). Not all items in a path are necessarily IElements. Furthermore. implementing IElement is not a requirement for entities, so you can't do better than object I guess.

Reply:
You are right. I missed that.
Changed to 'object'.